test/no_diff.jl

```diff
@@ -0,0 +1,52 @@
+@testset EncasedTestSet "wrapper" begin
+    global output
+    output = @capture_out begin
+        try
+            @testset ExtendedTestSet "matrices" begin
+                @test [1 2; 3 4] == [1 4; 3 4]
+             end
+        catch
+        end
+    end
+end
+@test contains(output, "Diff:\nnothing")
```

Comment:
this one's weird. It's been a while since I've looked at this package so I don't remember what the expected behavior is here. It doesn't seem like `Diff: nothing` would be what you'd want though, given that there is a difference. If Matrices aren't supported by DeepDiffs, I'd think this just wouldn't include the `Diff:` at all.

probably worth a comment to explain.

Reply:
Yeah. Maybe the better approach is 
1. disable diffing on matrices for now 
2. check out what DeepDiffs is up to
3. update to include matrix functionality

Reply:
okay, the issue was more interesting than I thought -- there is an implicit `else` branch that led to a `nothing` in the logic.